programmers/문자열-내-p와-y의-개수-세기.js

```diff
@@ -0,0 +1,33 @@
+function solution(s){
+    var count_p = 0;
+//s내부의 p의 개수를 세줄 변수를 선언하였습니다.    
+    var search_p = 'p';
+// indexof라는 함수를 이용하여 p를 찾기 위하여 search_p라는 문자 p를 담고있는 변수를 선언하였습니다. 
+    var index_p = s.indexOf(search_p);
+// indexof라는 함수를 사용하여 s내부의 p의 위치를 입력받는 index_p를 선언하였습니다.
+    while(index_p !== -1){
+        count_p ++;
+        index_p = s.indexOf(search_p,index_p+1);
+    }
+/*index_p가 문자열 s의 마지막인덱스 까지도는 동안나오는 p의 개수를 count_p에 누적시키고,
+ indexof함수는 searchvalue가 없게 되면 -1을 리턴하므로 while문을 빠져나와 줍니다.*/
+    var count_y = 0;
+    var search_y = 'y';
+    var index_y = s.indexOf(search_y);
+    while(index_y !== -1){
+        count_y ++;
+        index_y = s.indexOf(search_y,index_y+1);
+    }
+ //문자 s에 대해서도 같은 방식으로 진행   
+    if(count_p === count_y){
+        return true;
+    }
+    else{
+        return false;
+    }
+}
```

Comment:
```suggestion
    if(count_p === count_y) return true;
    
    return false
}
```
 if문에서 return을 쓰면 else를 쓰지 않아도 밖에서 그 외 조건을 return합니다 또한 조건문이 한줄이라면  이렇게  
괄호를 생략할 수 있습니다  

삼항 연산자를 사용하면 이렇게도 좀 더 간결하게  코드를 작성 할 수 있습니다  다만 삼항 연산자는 코드의 길이를 
줄여주지만  상황에 따라서 오히려 가독성이 안좋아질 수 있기 때문에  적절한 상황에만 사용 해주시는 게 좋습니다 
`return count_p === count_y ? true : false`